app/controllers/admin/categories_controller.rb

```diff
@@ -2,6 +2,9 @@ class Admin::CategoriesController < ApplicationController
 
   before_filter :ensure_is_admin
 
+  #Allow admin to access admin panel before email confirmation
+  skip_filter :cannot_access_without_confirmation
```

Comment:
@ovan yea, perhaps the AdminBaseController could be good one. Not pointing so directly to only pages or area, as the base controller could handle responsibilities also not directly related to certain pages or areas, but more just admin actions etc.. I'll switch to that.